config/environment.rb

```diff
@@ -3,3 +3,13 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+ActionMailer::Base.smtp_settings = {
```

Comment:
first of all, this setting should probably sit in production.rb.. secondly, we should probably use environment variable for username/password